lib/elixir/test/elixir/macro_test.exs

```diff
@@ -620,6 +621,57 @@ defmodule MacroTest do
     assert Macro.unpipe(quote(do: foo |> bar |> baz)) == quote(do: [{foo, 0}, {bar, 0}, {baz, 0}])
   end
 
+  # guard
+
+  test "guard" do
```

Comment:
(Incidentally I'm not sure if it's idiomatic to wrap code in error messages in backticks in Elixir, but this shows why I normally do: it helps it jump out in pasted stacktraces. Will remove on request.)